src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    top: 80vmin / 2 - 20vmin;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    animation: roll 3600s linear 60 forwards;
+    transform: rotate(30deg);
+    transform-origin: bottom;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    top: 80vmin / 2 - 38vmin;
```

Comment:
Similar to the previous comment, the calculation 'top: 80vmin / 2 - 38vmin;' might not work as intended. Use parentheses to ensure the correct order of operations: 'top: (80vmin / 2) - 38vmin;'.